Added pkg/schemas/jsonschemas to gitignore

shunr-hpe opened the pull request:
Changed the unit test to remove pkg/schemas/jsonschemas when done.

davidallendj reviewed (commented):
I believe the `pkg/schemas/jsonschemas` directory is generated from the test right? I don't think it's necessary to add generated data to the`.gitignore` and we should probably remove it in the test itself instead.

shunr-hpe committed:
Changed test to remove pkg/schemas/jsonschemas when done

Signed-off-by: Shane Unruh <shane.unruh@hpe.com>

shunr-hpe commented:
@davidallendj Thanks for the suggestion.

I changed the test to remove the directory, and removed my changes to gitignore and Makefile.

My goal is to not have a dirty `git status` after running the makefile targets.

davidallendj reviewed (approved):
LGTM
